code/src/components/BrandSelectOption.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+
+export const BrandSelectOption = ({ brand, setBrand}) => {
+
+    const onBrandChange = (e) => {
+        setBrand(e.target.value)
+    }
+    
+    return(
+        <>
+            <div className="select-container">
+                <label htmlFor="brand" className="option-label" tabIndex="0">Favourite Sneaker Brand: </label>
+                <select id="brand" value={brand} onChange={onBrandChange} tabIndex="0" required>
+                    <option disabled></option>
+                    <option>Nike</option>
+                    <option>Jordan</option>
+                    <option>Adidas</option>
+                    <option>Reebok</option>
+                    <option>New Balance</option>
+                    <option>Karhu</option>
```

Comment:
Another way to do this would have been to store the different brands in an array and then map them here 🤷‍♂️